hera_pspec/pspecdata.py

```diff
@@ -681,6 +685,75 @@ def get_G(self, key1, key2):
                 G[i,j] += np.einsum('ab,ba', iR1Q[i], iR2Q[j])
 
         return G / 2.
+
+    def get_H(self, key1, key2, taper='none'):
```

Comment:
I don't see `H_ab` defined anywhere in the note?

Reply:
Actually, it's there. Equation 33. Unfortunate page break location makes it hard to see.